src/fortran_routing/diffusive_v02frwk/diffusive_cnt/diffusive_cnt.f90

```diff
@@ -252,8 +252,8 @@ subroutine diffnw(dtini_g, t0_g, tfin_g, saveinterval_ev_g, dt_ql_g, dt_ub_g, dt
         end do
 
         ! +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
-		!  interpolation of boundaries at the initial time step
-		! +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+	!  interpolation of boundaries at the initial time step
+	! +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
```

Comment:
I think these are indentation changes. When I look at the revised code through GitHub, the indentation is still questionable. Let's make sure that the indentation and alignment are tight to facilitate readability.